### PR TITLE
Feat/room model

### PR DESCRIPTION
**Pull request by mokshgrappus007**
added room model and did some code formating

### CODE REVIEW

**Comment by noorbhatia10** [comment]
@mokshgrappus007 , we're deprioritizing room creation for now, it will just add complexity. If everything works in a single session then room creation will be easier

**Comment by mokshgrappus007** [comment]
haven't added any logic for room creation it's just whatever we are storing in SessionState like currentPlayerId, Pointslist, playerlist for that i've made a different model only i.e RoomModel, to remove code complexities and also did some code cleanup

**Comment by noorbhatia10** [comment]
I don't think we need an additional model, sessionstate already is kind of a model

**Comment by mokshgrappus007** [comment]
ok then i'll remove the model

**Comment by mokshgrappus007** [comment]
ok removed the model @noorbhatia10

File: pubspec.yaml [hunk 1]
```diff
@@ -21,6 +21,7 @@ dependencies:
   intl: ^0.18.0
   player_repository:
     path: api/packages/player_repository
+  shared_preferences: ^2.2.0
```
Comment (sarthak-grappus) [inline pubspec.yaml]: hatade

**Pull request closed without merging**